Docs: Readd old image documentation (#3191)
cobbler/cobbler: docs/cobbler.rst

Cobbler image
=============

The primary and recommended use of cobbler is to deploy systems by building them, as though from the OS manufacturer's
distribution, e.g Redhat kickstart. This method is generally easier to work with and provides an infrastructure which is
not only more sustainable but also much more flexible across varieties of hardware.

But Cobbler can help with image-based booting, physically and virtually. Some manual use of other commands beyond what
is typically required of cobbler may be needed to prepare images for use with this feature, and the usage of these
commands varies substantially by the type of image.

There is a lot more to go here. For the moment we just have 1 example of using the "memdisk" image type

Example:

.. code-block:: shell

    $ cobbler image

memdisk - Oracle / Sun Maintenance CD
-------------------------------------

The 'memdisk' image type can be used to PXE boot Oracle / Sun maintenance CDs.
`Their manual <http://docs.oracle.com/cd/E19121-01/sf.x2250/820-4593-12/AppB.html#50540564_72480>`_ gives details on how

> to copy the boot CD to a PXE server. The procedure is even easier with cobbler since the system takes care of most of
> that stuff for you.

Reviewer: ```suggestion
to copy the image from a CD to a PXE server. The procedure is even easier with Cobbler since the system takes care of most of
it for you.
```

Take your ISO for the boot CD and mount is as a loopback mount somewhere on your cobbler server, and then copy the
``boot.img`` file out into your tftpboot directory. Then add an image of type ``memdisk`` which uses it. As of this

> writing the following command line command will fail due to a known bug, but the web interface can successfully be used

Author: I don't know but I will clarify this with another PR in the future.

Memtest
-------

If installed, cobbler will put an entry into all of your PXE menus allowing you to run memtest on physical systems
without making changes in Cobbler. This can be handy for some simple diagnostics.

Steps to get memtest to show up in your PXE menus:

.. code-block:: shell

   $ zypper/dnf install memtest86+
   $ cobbler image add --name=memtest86+ --file=/path/to/memtest86+ --image-type=direct
   $ cobbler sync

Targeted Memtesting
-------------------

However, if you already have a cobbler system record for the system, you can't get the menu. No problem!

.. code-block:: shell

   cobbler image add --name=foo --file=/path/to/memtest86 --image-type=direct
   cobbler system edit --name=bar --mac=AA:BB:CC:DD:EE:FF --image=foo --netboot-enabled=1

The system will boot to memtest until you put it back to it's original profile.

.. warning:: When restoring the system back from memtest, make sure you turn it's netboot flag **off** if you have it
             set to PXE first in the BIOS order, unless you want to reinstall the system!

.. code-block:: shell

   $ cobbler system edit --name=bar --profile=old_profile_name --netboot-enabled=0

> Naturally if you **do** want to reinstall it after running memtest, just use ``--netboot-enabled=true``.

Reviewer: ```suggestion
If you do want to reinstall it after running memtest, use ``--netboot-enabled=true``.
```
